CMakeLists.txt

```
@@ -17,6 +17,11 @@ option(FORCE_BUILD_VENDOR_PKG
   "Build console_bridge from source, even if system-installed package is available"
   OFF)
 
+# Avoid DOWNLOAD_EXTRACT_TIMESTAMP warning for CMake >= 3.24
+if (POLICY CMP0135)
```

```suggestion
if(POLICY CMP0135)
```

(this is why the Rpr job is failing)